### ISSUE TITLE
CI Build failing due to unsupported `let`

### ISSUE THREAD

**Issue by paulbellamy** [stellar/js-stellar-base#414]
**Describe the bug**

https://travis-ci.com/github/stellar/js-stellar-base/jobs/498317989
```
Firefox 39.0 (Windows 7) ERROR
  SyntaxError: let is a reserved identifier
  at test/unit/crypto_test.js:48:2
```

My hunch is that we are transpiling to a version which supports `let`, which is replacing the import on `test/unit/crypto_test.js:1` with `let crypto = ...`. But, firefox 39 [doesn't support `let`](https://caniuse.com/let).

**What version are you on?**

See CI failure

**To Reproduce**

See CI failure.

**Expected behavior**

A passing build. Maybe we should transpile to a target which doesn't use `let` for better compatibility?

**Additional context**

Found on https://github.com/stellar/js-stellar-base/pull/413

**Comment by Shaptic** [comment]
Not only is Firefox 39 absurdly old (I'm on v86), so is Windows 7; the tests definitely need to be updated.

**Comment by Shaptic** [comment]
I believe this was closed by #419.
